moab/meta.yaml

```diff
@@ -0,0 +1,17 @@
+package:
+  name: moab
+  version: 4.8.0
+
+source:
+  fn: moab.tar.gz
+  url: ftp://ftp.mcs.anl.gov/pub/fathom/moab-4.8.0.tar.gz
```

Comment:
It could be, but then that is super annoying because then the batlab nodes need git.  Is it safer to pull the tarball from the bitbucket tag?